mock-plugin/src/main/java/io/mosip/signup/plugin/mock/service/MockProfileRegistryPluginImpl.java

```diff
@@ -87,8 +87,8 @@ public void validate(String action, ProfileDto profileDto) throws InvalidProfile
         JsonNode inputJson = profileDto.getIdentity();
         List<String> requiredFields = action.equals("CREATE") ? requiredFieldsOnCreate : requiredFieldsOnUpdate;
         for (String fieldName : requiredFields) {
-            if (!inputJson.hasNonNull(fieldName) || (inputJson.get(fieldName).isArray() && inputJson.get(fieldName).isEmpty())) {
-                log.error("Null value found in the required field of {}, required: {}", fieldName, requiredFieldsOnCreate);
+            if (!fieldName.isEmpty() && (!inputJson.hasNonNull(fieldName) || (inputJson.get(fieldName).isArray() && inputJson.get(fieldName).isEmpty()))) {
```

Comment:
This change already exists in the current 'develop' branch. 
FYR https://github.com/mosip/esignet-plugins/blob/develop/mock-plugin/src/main/java/io/mosip/signup/plugin/mock/service/MockProfileRegistryPluginImpl.java#L90

was the latest update taken before raising the PR? @Md-Humair-KK